### DISCUSSION TITLE
Define a max amount in USDT

### DISCUSSION THREAD

**Discussion by HavaneFr** [cdalton713/trading-bot-new-coins#17] [Ideas]
Hi

Is it possible to define a max amount to buy in usdt, and not in quantify of QUOTE_TICKER ?
For exemple, buy 10 DAR not the same risk as buy 10 BNX

Thanks for your work

**Comment by cdalton713** [comment]
Hello,

If you want to purchase in USDT please keep QUOTE_TICKER as is: i.e.,  `QUOTE_TICKER: 'USDT'` 

This means the paired coin will be USDT and X QUANTITY will be purchased.  So in the example, with QUANTITY = 10 and QUOTE_TICKER = 'USDT', when any new coin comes out it will purchase $10 USDT's worth of that specific coin, so the risk is the same no matter which coin comes out.

Let me know if I'm misunderstanding your question!
Thanks
Christian

**Reply by HavaneFr** [reply]
Yes, perfect. Maybe update the comment "For example, BTCUSDT will be bought with pairing 'USDT'"

By "For example, a new coin NEW, will be bought with pairing 'NEWUSDT for a maximum amount  of QUANTITY USDT"
